testsuite/features/secondary/allcli_system_group.feature

```diff
@@ -118,3 +192,12 @@ Feature: Manage a group of systems
     Then I should see a "System group" text
     And I should see a "new-systems-group" text
     And I should see a "deleted" text
+
+  Scenario: Cleanup: regenerate search index for later tests
+    When I clean the search index on the server
+
+  Scenario: Cleanup: remove old packages
+    When I disable repository "test_repo_rpm_pool" on this "sle_minion" without error control
+    And I refresh the metadata for "sle_minion"
+    And I remove package "andromeda-dummy" from this "sle_minion" without error control
+    And I remove package "virgo-dummy" from this "sle_minion" without error control
```

Comment:
please add missing linefeed on last line

(there's probably some setting to fix in your text editor)